_layouts/default.html

```diff
@@ -42,44 +42,43 @@
-            <div class="navbar-header">
-                <button type="button" class="navbar-toggle" data-bs-toggle="collapse" data-bs-target="#bs-example-navbar-collapse-1">
-                    <span class="sr-only">Toggle navigation</span>
-                    {% if page.url == '/' or page.url == '/index.html' %}
-                      <span class="icon-bar white-icon-bar"></span>
-                      <span class="icon-bar white-icon-bar"></span>
-                      <span class="icon-bar white-icon-bar"></span>
-                    {% else %}
-                      <span class="icon-bar"></span>
-                      <span class="icon-bar"></span>
-                      <span class="icon-bar"></span>
-                    {% endif %}
-                </button>
-                <a class="navbar-brand" href="{{ site.baseurl }}/index.html">
-                    <img id="jupyter-nav-logo" class="navbar-logo" src="{{ site.baseurl }}/assets/nav_logo.svg" alt="Jupyter logo">
-                </a>
-            </div>
-            <div class="blur"></div>
-            <!-- Collect the nav links, forms, and other content for toggling -->
-            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
-                <ul class="nav navbar-nav navbar-right">
-                    {%- for nav in layout.nav -%}
-                    <li>
-                        <a href="{{nav.url}}"
-                           {%- if nav.newpage %}target="_blank" rel="noopener noreferrer"{% endif %}
-                           class="tab {% if nav.url == page.url %}navbar-active{% endif %}">
-                           {{nav.title}}
-                       </a>
-                    </li>
-                    {%- endfor %}
-                </ul>
-            </div>
+    <nav class="navbar" role="navigation">
+        <div class="navbar-header">
+            <a class="navbar-brand" href="/">
+               <picture>
+                <source media="(max-width: 767px)" srcset="/assets/logos/logomark-orangebody-greyplanets.svg">
+                <source media="(min-width: 768px)" srcset="/assets/logos/rectanglelogo-greytext-orangebody-greymoons.svg">
+                <img class="navbar-logo"
+                     src="/assets/logos/rectanglelogo-greytext-orangebody-greymoons.svg"
+                     alt="Jupyter logo"
+                     height=40
+                     loading="eager" />
+               </picture>
+            </a>
+        </div>
+        <button type="button"
+                class="hamburger"
+                data-bs-toggle="collapse"
+                data-bs-target="#bs-navbar-collapse" />
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar white-icon-bar"></span>
+            <span class="icon-bar white-icon-bar"></span>
+            <span class="icon-bar white-icon-bar"></span>
+        </button>
+        <div class="mobile-clearfix"></div>
```

Comment:
what does this do?

Reply:
The clearfix kicks the <ul> with the floating page links down below the logo block with a `clear:both`. There is almost certainly another way to do this. I was sticking with the same method used by the existing nav.